acceptance-tests/src/test/scala/ch/datascience/graph/acceptancetests/tooling/RDFStore.scala

```diff
@@ -131,11 +130,9 @@ object RDFStore {
     jenaReference.read
       .map { jena =>
         val queryResults = jena.connection
-          .query("""
-                   |SELECT ?s ?p ?o WHERE {
-                   |  ?s ?p ?o .
-                   |}
-                   |""".stripMargin)
+          .query("""|SELECT ?s ?p ?o 
+                    |WHERE { ?s ?p ?o }
+                    |""".stripMargin)
```

Comment:
I know this is just for tests but maybe putting a `LIMIT` there would make the query look less scary?

Reply:
Fixed